sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -642,6 +642,15 @@ object SQLConf {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(0L)
 
+  val ADAPTIVE_EXPAND_PARTITIONS_ENABLED =
```

Comment:
We don't need a config, as people can exclude any optimizer rule by name.

Reply:
we cann't do this, currently only AQE Optimizer support exclude rule

Reply:
OK, then we need to think about the config name.

How about `spark.sql.adaptive.optimizeSkewsInRebalacePartitions`?

Reply:
updated